llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -2919,10 +2998,19 @@ gentbl(
     ],
 )
 
+cc_library(
+    name = "llvm-objcopy-headers",
+    hdrs = glob(["tools/llvm-objcopy/**/*.h"]),
+    strip_include_prefix = "tools/llvm-objcopy",
+)
+
 cc_binary(
     name = "llvm-objcopy",
     srcs = glob([
         "tools/llvm-objcopy/**/*.cpp",
+        # Note that we redundantly include the headers here to allow files to
```

Comment:
But here you don't need to turn off header_modules?

Reply:
Ah, probably do here as well because they're duplicated, yeah.